client/packages/invoices/src/utils.ts

```diff
@@ -192,7 +192,9 @@ export const inboundLinesToSummaryItems = (
     createSummaryItem(itemId, lines)
   );
 };
-export const canDeleteInvoice = (invoice: OutboundRowFragment): boolean =>
+export const canDeleteInvoice = (
+  invoice: OutboundRowFragment | PrescriptionRowFragment
+): boolean =>
```

Comment:
Why was this needed?

Reply:
I think it must have been a merge thing... Both outbound and prescriptions use this method to check if a prescription can be deleted, so need both fragments